fhf-datafangst-client/src/components/MyPage/MyFollowers.tsx

```diff
@@ -0,0 +1,139 @@
+import { PersonAdd, PersonRemove } from "@mui/icons-material";
+import { Box, Grid, IconButton } from "@mui/material";
+import { VesselFilter } from "components/Filters/VesselFilter";
+import { Vessel } from "generated/openapi";
+import React, { FC } from "react";
+import {
+  useAppDispatch,
+  useAppSelector,
+  selectUser,
+  selectUserVessels,
+  setSelectedVessels,
+  updateUser,
+  selectVesselsByFiskeridirId,
+} from "store";
+import { VesselInfo } from "./VesselInfo";
+import { useAuth } from "oidc-react";
+
+interface ContentProps {
+  vessels: Vessel[];
+}
+
+export const FollowList: FC = () => {
+  const { userData } = useAuth();
+  const dispatch = useAppDispatch();
+  const vessels = useAppSelector(selectVesselsByFiskeridirId);
+  const user = useAppSelector(selectUser);
+  const selectedVessels = useAppSelector(selectUserVessels);
+
+  const followList = user?.following.map(
+    (fiskeriDirId) => vessels[fiskeriDirId],
+  );
+
+  const Content: FC<ContentProps> = (props: ContentProps) => {
```

Comment:
This is large enough to just be its own component (with a better name)

Reply:
I agree, that makes more sense